### ISSUE TITLE
Specify the version for which written documentation?

### ISSUE THREAD

**Issue by kulakowka** [senecajs/senecajs.org#106]
It would be nice if for each example or documentation for API was indicated version of the library. The usual case where documentation was not updated with new versions of the library. Maybe you can specify which version of the library has been written documentation?

**Comment by kulakowka** [comment]
Also, it is possible to add the current version of the library. This is easily done by using badges (http://badge.fury.io/for/js/seneca). 

For example: [![npm version](https://badge.fury.io/js/seneca.svg)](https://badge.fury.io/js/seneca)

```markdown
[![npm version](https://badge.fury.io/js/seneca.svg)](https://badge.fury.io/js/seneca)
``` 

This badge can be placed near to the indication of the version in the documentation. This allows you to quickly determine what documentation outdated and does not make mistakes.

**Comment by AdrieanKhisbe** [comment]
@kulakowka 
Where do you want the badge to be?
(badges are currently being added to all seneca-plugins README)

**Comment by mcdonnelldean** [comment]
I'd like to do this but it would need to be done at a content level. I think when 1.0 hits we should use the opportunity to version the docs by url. Thoughts @senecajs/contributors

**Comment by AdrieanKhisbe** [comment]
hapijs website could be an example how to do that

**Comment by mcdonnelldean** [comment]
Closed as merge of #156
